announcing-pydoc-io.rst

```diff
@@ -0,0 +1,93 @@
+
+Announcing pydoc.io beta
+========================
+
+Today we're excited to announce our latest project: https://pydoc.io.
+This work was made possible by the :doc:`MOSS Grant </rtd-awarded-mozilla-open-source-support-grant>`_ from Mozilla.
+Thanks to Mozilla for giving us the money to set aside time to build this wonderful community resource.
+
+About pydoc.io
+--------------
+
+Running Read the Docs,
+we've always been proud of the documentation tooling that the Python community has.
+We prioritize prose over API documentation listings,
+and generally have a high standard of documentation in our projects.
+
+However,
+there are a specific set of use cases that API reference documentation support,
+and the Python community doesn't support them well.
+Tools like http://godoc.org and http://rubydoc.info provide these for others languages,
+and we now introduce http://pydoc.io as the Python communities version of that.
+
+What is it?
+-----------
+
+Pydoc.io will eventually auto-generate API reference documentation for every package on PyPI.
+Our beta release will support a limited set of packages that we are using for testing.
+Over time we'll expand the tooling to automatically support generating docs for any package posted on PyPI.
+
+Reference documentation is wonderful for folks who already know how a library works,
+and simply want to use it.
+Giving them simple access to nicely rendered API documentation supports using and understanding libraries in a nicer way than reading code.
+We're happy to build out these tools,
+along with the wonderful prose tools we've always had with Sphinx itself.
+
+Our progress and some context is tracked in `issue 1957 <https://github.com/rtfd/readthedocs.org/issues/1957>`_.
+
+The tools
+---------
+
+This project is built on top of `Sphinx`_ at it's core.
+On top of that,
+we've layered a few different tools:
+
+* `sphinx-autoapi`_ which is the tool that takes Python source files and turns them into RST.
+  * The `pydocstyle`_ AST parser, which we use for a parse-only representation of Python classes.
+* `pydoc.io`_ is a Django application that is the actual web front-end and building code.
+* `sphinx-apitheme`_ is the Sphinx theme that powers the actual API listing pages.
+
+The biggest piece to call out here is the parse-only documentation generation.
+Sphinx's autodoc module,
+and most Python documentation generators rely on importing the code to get docstrings.
+As we've learned with Read the Docs,
+this is quite impractical for a large set of libraries,
+including the scientific ecosystem that heavily relies on C libraries.
+**Building and supporting a parse-only API tool makes doc builds faster and more reliable.**
+
+We have built on top of the ``pydocstyle`` AST currently,
+but it looks like we'll likely have to build out our own AST traversal to get all the information we need.
+We have `looked <https://github.com/davidhalter/jedi/issues/630>`_ into some of the other AST options in Python,
```

Comment:
Odd linking here, feel like this should be wrapped around `some of the other AST options in Python`. Additionally, `some of the other AST options in Python` could be clearer/more specific. `into unified solutions for AST traversal of docstrings` or whatever.